### DISCUSSION TITLE
Murata 1YN (Cypress 43439)

### DISCUSSION THREAD

**Discussion by victorallume** [micropython/micropython#14251] [Libraries & Drivers]
I've been playing with getting the Murata 1YN wifi module working with micropython. It's a Cypress 43439 chipset internally, and a meant to a pin compatible replacement for the 1DX. I tried to create the relevant firmware files in the `lib/cyw43-driver/firmware` folder from originals at https://github.com/murata-wireless/cyw-fmac-fw/ but couldn't get it to work. But then I tried the stock 43439 firmware in micropython, and voila, it works fine. 

Is it possible there are some configuration changes that might need tweaking in the wifi_nvram file, or that the CLM blob might not be matched for the module?

**Comment by jotamudo** [comment]
I didn't write the micropython driver for it, but I'm pretty sure you'd have to adapt all the calls on the micropython side to encompass the new firmware custom commands (think IOCTL and such)
